### ISSUE TITLE
ec2.Vpc.from_lookup return resource that does not implement IVpc

### ISSUE THREAD

**Issue by dxunix** [aws/aws-cdk#5040]
Try to get a VPC object from the existing VPC through ec2.Vpc.from_lookup() by vpc_id.  However, the returned object does not have the attributes described in the document.

### Reproduction Steps
 ```python       
        vpc = ec2.Vpc.from_lookup(self, "myvpc", vpc_id=VPC_ID)
        # _vpc = ec2.Vpc.from_lookup(self, "myvpc", is_default=True)
        print("vpc is {}".format(vpc.vpc_id))
```

### Error Log
```
Traceback (most recent call last):
  File "app.py", line 11, in <module>
    EcsTestStack(app, 'ecs-test-stack', env={'account': '************', 'region': 'us-east-1'})
  File "/Users/dye/dev/cdk/service-chk/.env/lib/python3.7/site-packages/jsii/_runtime.py", line 66, in __call__
    inst = super().__call__(*args, **kwargs)
  File "/Users/dye/dev/cdk/service-chk/ecstest/ecstest_stacks.py", line 28, in __init__
    print("vpc is {}".format(vpc.vpc_id))
AttributeError: 'Resource' object has no attribute 'vpc_id'
```

### Environment

  - **CLI Version      :1.16.3 (build 8c7f593)**
  - **Framework Version:**
  - **OS               :MacOS (Darwin ********** 18.7.0 Darwin Kernel Version 18.7.0: Sat Oct 12 00:02:19 PDT 2019; root:xnu-4903.278.12~1/RELEASE_X86_64 x86_64)**

  - **Language         :Python**

### Other

--- 

This is :bug: Bug Report

**Comment by rix0rrr** [comment]
This is a known jsii issue. I will kick it over to @RomainMuller who can tell you when the fix will be released.

**Comment by rix0rrr** [comment]
This should have been fixed in 1.16.3. Can you make sure that your framework libraries are that version as well (not just the CLI) and try again?

**Comment by dxunix** [comment]
updated aws-cdk.core,  it is working now.  Thank you!
